specs/_features/maxeb_increase/capella.py

```diff
@@ -1840,6 +2012,40 @@ def process_voluntary_exit(state: BeaconState, signed_voluntary_exit: SignedVolu
     initiate_validator_exit(state, voluntary_exit.validator_index)
 
 
+def process_consolidation(state: BeaconState, signed_consolidation: SignedConsolidation) -> None:
+    assert(len(state.pending_consolidations) < PENDING_CONSOLIDATIONS_LIMIT)
+    consolidation = signed_consolidation.message
+    target_validator = state.validators[consolidation.target_index]
+    source_validator = state.validators[consolidation.source_index]
```

Comment:
I guess it's worth making sure that these are two different validators
```suggestion
    consolidation = signed_consolidation.message
    assert(consolidation.target_index != consolidation.source_index)
    target_validator = state.validators[consolidation.target_index]
    source_validator = state.validators[consolidation.source_index]
```

Reply:
Checked in the canonical PR https://github.com/ethereum/consensus-specs/pull/3618